src/common/components/Abilities/AbilityRow/AbilityRow.tsx

```diff
@@ -0,0 +1,35 @@
+import classnames from "classnames/bind";
+import React from "react";
+import { calculateModifierBonus } from "../../../Utilities";
+import { Ability, Modifier } from "../../../models";
+import { CharacterSelector } from "../../../redux/Character.slice";
+import { useSelector } from "../../../redux/reduxHooks";
+import styles from "./AbilityRow.module.css";
+
+const cx = classnames.bind(styles);
+const { useState, useEffect } = React;
+
+interface AbilityRowProps {
+  ability: Ability;
+  proficiencyBonus: number;
+  modifier: Modifier;
```

Comment:
Notice how I am passing modifier and proficiency bonus. They are all part of base character and I could import base character but I want to try and only import it into the top level component when possible.